recipes/coost/all/conanfile.py

```diff
@@ -0,0 +1,97 @@
+from conan import ConanFile
+from conan.errors import ConanInvalidConfiguration
+from conan.tools.build import check_min_cppstd
+from conan.tools.cmake import CMake, CMakeDeps, CMakeToolchain, cmake_layout
+from conan.tools.files import copy, get
+from conan.tools.microsoft import is_msvc, is_msvc_static_runtime
+import os
+
+required_conan_version = ">=1.50.0"
+
+
+class CoostConan(ConanFile):
+    name = "coost"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/idealvin/coost"
+    license = "MIT"
+    description = "A tiny boost library in C++11."
+    topics = ("coost", "coroutine", "c++11")
```

Comment:
```suggestion
    topics = ("coroutine", "cpp11", "boost")
```
Topics are keywords related to the project to be used in a search. Conan and the project name are not good words. Also, avoid signals like `+` to skip search engine limitations.